tttx/online/R3BTttxOnlineSpectra.h

```diff
@@ -0,0 +1,139 @@
+/******************************************************************************
+ *   Copyright (C) 2019 GSI Helmholtzzentrum für Schwerionenforschung GmbH    *
+ *   Copyright (C) 2019-2024 Members of R3B Collaboration                     *
+ *                                                                            *
+ *             This software is distributed under the terms of the            *
+ *                 GNU General Public Licence (GPL) version 3,                *
+ *                    copied verbatim in the file "LICENSE".                  *
+ *                                                                            *
+ * In applying this license GSI does not waive the privileges and immunities  *
+ * granted to it by virtue of its status as an Intergovernmental Organization *
+ * or submit itself to any jurisdiction.                                      *
+ ******************************************************************************/
+
+// ------------------------------------------------------------
+// -----               R3BTttxOnlineSpectra               -----
+// -----    Created 23/01/24 by M.Whitehead               -----
+// -----           Fill TTT10 online histograms           -----
+// ------------------------------------------------------------
+
+#ifndef R3BTttxOnlineSpectra_H
+#define R3BTttxOnlineSpectra_H 1
```

Comment:
#pragma once
instead of 
#ifndef R3BTttxOnlineSpectra_H
#define R3BTttxOnlineSpectra_H 1